### ISSUE TITLE
Advanced Bones

### ISSUE THREAD

**Issue by PhantomKobra** [studio-minus/ppg-bugs-and-requests#2204]
**Feature description**
Bones are a feature in every human body; without them, we would be only be bags of flesh and muscle held together by nothing. Humans in PPG are no exception to this, yet, the bones in their body seem to just be decoration rather than actual structural support. What I was thinking is that human bones actually have destruction to their names. Yes, bones break when enough pressure is applied to them, but can you see that when you open up inside and have a look for yourself? No, you can't. What I was thinking is that bones actually have advanced physics to their name; if a bone breaks and you burn away all of the skin, you can see the exact fracture in the exact bone that you broke. Bones will also snap and break from gunshots, which causes different types of fractures depending on the caliber, bullet size and speed. Open fractures will happen from serious injures (falling from great heights) and they will snap and bulge through the skin, sometimes breaking through it entirely. A lot can be said here, but this is a feature I'd like to see in-game.

**Feature purpose**
Do I really need to explain this? 
1. Break every single bone in a human
2. Use the thrusterbed, burn away all of the flesh and muscle on the human to reveal the skeleton

I don't see any break in the bone AT ALL. They're just decorations right now to make the humans look realistic. The same could be said about the other parts of the body, and yes, it's damn hard to code all of that, but maybe adding the smaller specifics could add just a hint of detail into the body, making it seem realistic rather than being there for the sake of being there

**Additional context**

**Comment by PhantomKobra** [comment]
After playing the game for a couple more minutes, I just realized I'm a dumbass.
Thank you, Dindin.

**Comment by bibbybabby** [comment]
I'd also like to see the bones heal by itself because when they break they break and the bone will never heal by itself

**Comment by DindinYT37** [comment]
> I'd also like to see the bones heal by itself because when they break they break and the bone will never heal by itself

Yeah but bones usually take around 6 to 12 weeks to heal, so don't know about that one if we're talking about realistic.

**Comment by DindinYT37** [comment]
> > > I'd also like to see the bones heal by itself because when they break they break and the bone will never heal by itself
> > 
> > 
> > Yeah but bones usually take around 6 to 12 weeks to heal, so don't know about that one if we're talking about realistic.
> 
> Wounds also take a week to heal, but in the game they heal in a minute, your point?

Wounds are not equal to bones.

**Comment by bibbybabby** [comment]
> > > > I'd also like to see the bones heal by itself because when they break they break and the bone will never heal by itself
> > > 
> > > 
> > > Yeah but bones usually take around 6 to 12 weeks to heal, so don't know about that one if we're talking about realistic.
> > 
> > 
> > Wounds also take a week to heal, but in the game they heal in a minute, your point?
> 
> Wounds are not equal to bones.

So? It's still gameified

**Comment by bibbybabby** [comment]
> > > > > > I'd also like to see the bones heal by itself because when they break they break and the bone will never heal by itself
> > > > > 
> > > > > 
> > > > > Yeah but bones usually take around 6 to 12 weeks to heal, so don't know about that one if we're talking about realistic.
> > > > 
> > > > 
> > > > Wounds also take a week to heal, but in the game they heal in a minute, your point?
> > > 
> > > 
> > > Wounds are not equal to bones.
> > 
> > 
> > So? It's still gameified
> 
> So what did you not get about "so don't know about that one **if we're talking about realistic**" exactly?

Realistic as in "the game feels more realistic" and not "realistic time"

**Comment by DindinYT37** [comment]
> > > > > > > I'd also like to see the bones heal by itself because when they break they break and the bone will never heal by itself
> > > > > > 
> > > > > > 
> > > > > > Yeah but bones usually take around 6 to 12 weeks to heal, so don't know about that one if we're talking about realistic.
> > > > > 
> > > > > 
> > > > > Wounds also take a week to heal, but in the game they heal in a minute, your point?
> > > > 
> > > > 
> > > > Wounds are not equal to bones.
> > > 
> > > 
> > > So? It's still gameified
> > 
> > 
> > So what did you not get about "so don't know about that one **if we're talking about realistic**" exactly?
> 
> Realistic as in "the game feels more realistic" and not "realistic time"

So two questions. First, why do you decide what I meant with realistic? Second, I was talking about realistic - not feels more realistic or realistic timing, it's obviously generally more realistic if the timing is also realistic.

**Comment by bibbybabby** [comment]
> > > So what did you not get about "so don't know about that one **if we're talking about realistic**" exactly?
> > 
> > 
> > Realistic as in "the game feels more realistic" and not "realistic time"
> 
> So, two questions. First, why do you decide what I meant with realistic? Second, I was talking about realistic - not feels more realistic or realistic timing, it's obviously generally more realistic if the timing is also realistic.

Because the request meant realistic that way and you can apply the timing realism to anything: Why do the wounds heal in a minute? Why do people rot in 5 minutes instead of a week? Why do people drown in 30 seconds instead of 2 minutes?

**Comment by DindinYT37** [comment]
> > > > So what did you not get about "so don't know about that one **if we're talking about realistic**" exactly?
> > > 
> > > 
> > > Realistic as in "the game feels more realistic" and not "realistic time"
> > 
> > 
> > So, two questions. First, why do you decide what I meant with realistic? Second, I was talking about realistic - not feels more realistic or realistic timing, it's obviously generally more realistic if the timing is also realistic.
> 
> Because the request meant realistic that way and you can apply the timing realism to anything: Why do the wounds heal in a minute? Why do people rot in 5 minutes instead of a week? Why do people drown in 30 seconds instead of 2 minutes?

As far as I can read, realistic was used only to describe the fact that the skeleton layer of a human should be modified in relevance to his bone fractures/broken bones. Also, you still don't decide what I meant with realistic so it's kind-of weird that you're still on about that.

**Comment by DindinYT37** [comment]
> > > > > > So what did you not get about "so don't know about that one **if we're talking about realistic**" exactly?
> > > > > 
> > > > > 
> > > > > Realistic as in "the game feels more realistic" and not "realistic time"
> > > > 
> > > > 
> > > > So, two questions. First, why do you decide what I meant with realistic? Second, I was talking about realistic - not feels more realistic or realistic timing, it's obviously generally more realistic if the timing is also realistic.
> > > 
> > > 
> > > Because the request meant realistic that way and you can apply the timing realism to anything: Why do the wounds heal in a minute? Why do people rot in 5 minutes instead of a week? Why do people drown in 30 seconds instead of 2 minutes?
> > 
> > 
> > As far as I can read, realistic was used only to describe the fact that the skeleton layer of a human should be modified in relevance to his bone fractures/broken bones. Also, you still don't decide what I meant with realistic so it's kind-of weird that you're still on about that.
> > _Not to mention that this is another argument that's absolutely pointless as it poses no relevance. Though I guess I'll have to wait for you to decide what I meant with absolutely pointless too, right?_
> 
> I think you're just overwhelmed with something because you are making assumptions and thinking that I'm "slow" but yea it's just all an another silly argument war so we both need to not act dumb

I could say something to make this argument 20 comments longer but I think I'd rather not. Anyway, I'm not really overwhelmed and I didn't say you're slow or something but I guess this closes it. I recommend hiding your comments as off-topic too.

**Comment by ras09191** [comment]
Dev probably wont implement this
